src/ceph-volume/ceph_volume/util/disk.py

```diff
@@ -636,6 +636,8 @@ def human_readable_size(size):
     suffixes = ['B', 'KB', 'MB', 'GB', 'TB']
```

Comment:
I would be nice to add 'PB' to the list IMO and it would be better to change this list to tuple. But making any of these modifications aren't necessary to get this PR merged. So, feel free to skip it if you want to.